src/latex/LaTeXWriter.jl

```diff
@@ -107,6 +107,13 @@ const DOCUMENT_STRUCTURE = (
 )
 
 function render(doc::Documenter.Document, settings::LaTeX = LaTeX())
+    if isempty(doc.users.sitename) # otherwise, the latex compiler will terminate with a cryptic "There's no line here to end" error
```

Comment:
```suggestion
    if isempty(doc.user.sitename) # otherwise, the latex compiler will terminate with a cryptic "There's no line here to end" error
```

Reply:
oops :D